src/1week/06.js

```diff
@@ -1,46 +1,32 @@
-const shopping_cart = [];
-let shopping_cart_total = 0;
+let shopping_cart = [];
+const MINPRICEOFFREESHIPPING  = 20
```

Comment:
```suggestion
const MIN_PRICE_OFF_REE_SHIPPING  = 20
```
상수라서 다 대문자로 처리한건 좋네요!
다만, 대문자일 때는 띄어쓰는곳에 `_`를 추가해주시면 변수명을 읽기 편할 것 같아요